bot/exts/recruitment/talentpool/_cog.py

```diff
@@ -489,13 +490,22 @@ async def _nomination_to_string(self, nomination: Nomination) -> str:
         entries_string = "\n\n".join(entries)
 
         start_date = time.discord_timestamp(nomination.inserted_at)
+
+        thread = None
+
+        if nomination.thread_id:
+            thread = await get_or_fetch_channel(nomination.thread_id)
+
+        thread_jump_url = f'[Jump to thread!]({thread.jump_url})' if thread else "*Not created*"
```

Comment:
We should catch the error if the channel isn't found. Could write it something like this:

```suggestion
        thread_jump_url = "*Not created*"
        if nomination.thread_id:
            try:
                thread = await get_or_fetch_channel(nomination.thread_id)
            except discord.HTTPException:
                thread_jump_url = "*Not found*"
            else:
                thread_jump_url = f'[Jump to thread!]({thread.jump_url})'
```

or

```suggestion
        thread_jump_url = "*Not created*"
        if nomination.thread_id:
            with contextlib.suppress(discord.HTTPException):
                thread = await get_or_fetch_channel(nomination.thread_id)
                thread_jump_url = f'[Jump to thread!]({thread.jump_url})'
```

(neither tested)